app/controllers/api/v1/groups_controller.rb

```diff
@@ -49,7 +49,7 @@ def update
         end
 
         new_permissions = update_params.fetch(:permissions, []).map do |perm|
-          if !Permission::VALID_SYSTEM_WIDE_PERMISSIONS.include?(perm)
+          if !Permission::SYSTEM_WIDE_PERMISSIONS.include?(perm)
```

Comment:
How would you feel about creating a static method on the Permission class like `Permission.valid_system_wide_permission?(perm)`?

Reply:
I think that's a good idea. It will help clean up the code some. I'll add it in. Thanks!